dist/tools/vera++/check.sh

```diff
@@ -12,6 +12,10 @@ CURDIR=$(cd "$(dirname "$0")" && pwd)
 : "${WARNING:=1}"
 
 . "$RIOTBASE"/dist/tools/ci/changed_files.sh
+# tests/pkg_utensor/models/deep_mlp_weight.hpp is an auto-generated file
+# with lots of commas so T009 takes very long. Since it is auto-generated, just
+# exclude it.
+EXCLUDE='^(.+/vendor/|dist/tools/coccinelle/include|dist/tools/fixdep/fixdep.c|dist/tools/lpc2k_pgm/src|tests/pkg_utensor/models/deep_mlp_weight.hpp)'
```

Comment:
```suggestion
EXCLUDE='^(.+/vendor/|dist/tools/coccinelle/include|dist/tools/fixdep/fixdep.c|dist/tools/lpc2k_pgm/src|tests/pkg_utensor/models/*)'
```

We can actually exclude all the models here

Reply:
Done.